pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "poetry.masonry.api"
 
 [tool.poetry]
 name = "yapapi"
-version = "0.3.0"
+version = "0.3.1-alpha.0"
```

Comment:
I'd release it as `0.3.1`.

Reply:
Sure, let's make pre-release first for `0.3.1-alpha.0`, then bump the version to `0.3.1` and do a release.

Reply:
@filipgolem @azawlocki actually, if, despite my opinion to the contrary, we didn't update the library itself, imvho, it didn't even require a release ...